src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
-    StreamHandle(ConstructorAccess ca, std::shared_ptr<Loop> ref)
-        : Handle{ca, std::move(ref)}
+    StreamHandle(typename Handle<T, U>::ConstructorAccess ca, std::shared_ptr<Loop> ref)
+        : Handle<T, U>{std::move(ca), std::move(ref)}
```

Comment:
`
../../swdt/./3rdparty/uvw/uvw/stream.hpp:158:36: error: expected ')' before 'ca'

     StreamHandle(ConstructorAccess ca, std::shared_ptr<Loop> ref)

                                    ^~

../../swdt/./3rdparty/uvw/uvw/stream.hpp:160:5: error: expected unqualified-id before '{' token

     {}

     ^
`
gcc can't resolve **ConstructorAccess** and **Handle** types. 
Look, in non WIN32 version of code you use **Handle<T, U>::Handle** ,  not simple **Handle**